python/ray/tune/trial.py

```diff
@@ -137,96 +153,34 @@ def __init__(self,
     def generate_id(cls):
         return binary_to_hex(random_string())[:8]
 
-    def start(self, checkpoint_obj=None):
-        """Starts this trial.
-
-        If an error is encountered when starting the trial, an exception will
-        be thrown.
+    def init_logger(self):
```

Comment:
This logger is also platform aware and should be able to extend. Maybe we can move this to TrialExecutor, or we could make another abstraction?

Reply:
Yep, `rsync` is not available in our production environment due to security issue